plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -335,7 +335,7 @@ function CoverMenu:updateItems(select_number)
                                     })
                                 end
                                 UIManager:close(self.file_dialog)
-                                DocumentRegistry:closeDocument(file)
+                                document:close()
```

Comment:
No issue if we're viewing the cover via History of the same currently opened document in Reader?
(I see that this is not the case, but I would have expected DocumentRegistry:closeDocument() to itself closes the doc when refs becomes 0 :/)

Reply:
Oh, ok, `Document:close()` does not really close it if refs is not 0 :) So, the cleverness it there instead of here, but same thing in the end.

Reply:
Hopefully DocumentRegistry does its job and returns the cached instance ^^.

Reply:
Document:close only actually tears down the instance if the refcount drops to zero, so, provided everything works out, it's safe.

Reply:
(Whee, missed the second comment ^^).

Yeah, it's more of a case of unintuitive naming & lack of documentation ;).